GameDefinition/Content/ItemKindOrgan.hs

```diff
@@ -206,7 +206,7 @@ screechingBeak = fist
   , idamage  = toDmg $ 2 `d` 1
   , iaspects = [Timeout $ 5 + 1 `d` 5]
   , ieffects = [Recharging $ Summon "scavenger" $ 1 + 1 `dl` 2]
-  , idesc    = ""
+  , idesc    = "Both a weapon and a warning."
```

Comment:
How about "beacon" or something? I'd need to check Wikipedia, but I think they make the calls more to attract buddies (and jakals, etc., also benefit from that) than to warn them against danger. Feel free to correct me though.